docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -1,61 +1,71 @@
+:::tip
+The transaction type hex value is output by any RPC method call which returns a transaction type, such as [`eth_getTransactionByHash`](https://ethereum.github.io/execution-apis/api-documentation/) for example.
+:::
 
-See the [EIP-712 transaction section below](#eip-712-transactions) for more information.
+### Legacy: `0x0`
 
-:::
+The transaction format used before the introduction of typed-transactions.
+
+### EIP-2930: `0x1`
 
-### When is a transaction considered final?
+The Ethereum Improvement Proposal [EIP-2930: Optional access lists](https://eips.ethereum.org/EIPS/eip-2930) addressed contract breakage risks introduced by EIP-2929. 
 
-**Transaction finality** refers to the promise that transactions cannot be reversed, altered, or mutated in the context of a blockchain network.
+Everything the legacy format has, plus an `accessList` parameter containing an array of addresses and storage keys.
 
-Under proof of stake, Ethereum transactions finalize in 2.5 epochs (16 minutes) on average under normal conditions: reverting that transaction would cost 1/3 of the total staked Ethereum.
+### EIP-1559: `0x2`
 
-Once a block has been filled and sealed in zk rollups, its state is committed to the main Ethereum chain. The proving stage is then started, and a SNARK validity proof is constructed for each block transaction. Once completed, the SNARK is sent for verification on the L1 smart contract, and the transaction state becomes final following verification.
+The Ethereum Improvement Proposal [EIP-1559: Fee market change for ETH 1.0 chain](https://eips.ethereum.org/EIPS/eip-1559) is an updated transaction type introduced in Ethereum's London fork. It addressed network congestion and excessive fees coming from bids. EIP-1559 transactions don't specify `gasPrice` and instead use a base fee which is adjusted by each block.
 
-From the standpoint of zkSync, _finality_ occurs when the transaction (the SNARK verification) is executed by L1. At this point, the guarantees are the same as any other L1 transaction within the same L1 block; the more L1 blocks issued after the initial block is processed, the less likely this transaction will be overturned.
+EIP-1559 transaction types contain everything from EIP-2930 and legacy transactions (apart from removing the `gasPrice`). 
 
-When a user transmits a transaction, zkSync currently waits for the entire block to be filled, which means the finality time may be longer depending on the volume of transactions sent via zkSync. The finality time will reduce as the throughput increases.
+Additional parameters added are the `maxPriorityFeePerGas` and `maxFeePerGas` where users can specific maximum fees they're willing to pay to prioritize their transactions.
 
-### What exactly are operators?
+:::important
+zkSync Era supports the EIP-1559 transaction-type format but does nothing with the max fee parameters as these are related to miner tips.
```

Comment:
Not only max fee, but tip as well. worth to confirm everything that is ignored @kailash-manasarovar

Reply:
By max fee parameters I presume it means both.
However,
> as these are related to miner tips.
`maxPriorityFeePerGas` has nothing to do with tips

Reply:
Yes, plural means both. And, I think you are not quite right @barakshani I have added some clarification about these two.

```
- `maxPriorityFeePerGas`: Is the maximum fee users are willing to give to miners as an incentive.
- `maxFeePerGas`: Is the maximum fee users are willing to pay in total. This includes the `maxPriorityFeePerGas` and network-determined base fee per gas.
```

Reply:
> Is the maximum fee users are willing to give to miners as an incentive.

incentive for what?
also, do you "give" or "pay" fee?

Reply:
users can specific maximum fees they're willing to pay to prioritize their transactions (sentence above) ... I think it's clear, no?
"pay" is good